articles/cosmos-db/create-mongodb-golang.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Azure Cosmos DB: Build a MongoDB API console app with Golang and the Azure portal | Microsoft Docs
+description: Presents a Golang code sample you can use to connect to and query Azure Cosmos DB
+services: cosmosdb
+documentationcenter: ''
+author: durgaprasad
+manager: ''
+editor: ''
+
+ms.assetid: 
+ms.service: cosmosdb
+ms.custom: quick start connect
+ms.workload: 
+ms.tgt_pltfrm: na
+ms.devlang: golang
+ms.topic: hero-article
+ms.date: 05/26/2017
+ms.author: durgaprasad
+
+---
+
+# Azure Cosmos DB: Build a MongoDB API console app with Golang and the Azure portal
+
+Azure Cosmos DB is Microsoft’s globally distributed multi-model database
+service. You can quickly create and query document, key/value, and graph
+databases, all of which benefit from the global distribution and horizontal
+scale capabilities at the core of Azure Cosmos DB.
+
+This quick-start demonstrates how to use an existing
+[MongoDB](https://docs.microsoft.com/en-us/azure/documentdb/documentdb-protocol-mongodb)
```

Comment:
Please update the link to https://docs.microsoft.com/en-us/azure/cosmos-db/mongodb-introduction